src/Components/Dashboard/Transactions/TransactionHistory.js

```diff
@@ -144,7 +144,7 @@ function TransactionHistory() {
       </div>
       )}
       {displayTransaction?.length ? (
-        <div className={styles.transaction__list__wrapper}>
+        <div style={{ marginBottom: '100px' }} className={styles.transaction__list__wrapper}>
```

Comment:
Please try to avoid inline css. Add marginBottom: 100px css to transaction__list__wrapper class

Reply:
Okay